src/repo.rs

```diff
@@ -802,6 +802,19 @@ impl Repository {
         }
     }
 
+    /// Create stream to write blob
```

Comment:
Could this copy some of the documentation from libgit2, including recommendations about what else to call and what sort of buffering this does?